drivers/overlay/overlay.go

```diff
@@ -2698,12 +2701,17 @@ func notifyReleaseAdditionalLayer(al string) {
 // redirectDiffIfAdditionalLayer checks if the passed diff path is Additional Layer and
 // returns the redirected path. If the passed diff is not the one in Additional Layer
 // Store, it returns the original path without changes.
-func redirectDiffIfAdditionalLayer(diffPath string) (string, error) {
+func redirectDiffIfAdditionalLayer(diffPath string, checkExistence bool) (string, error) {
 	if ld, err := os.Readlink(diffPath); err == nil {
 		// diff is the link to Additional Layer Store
 		if !path.IsAbs(ld) {
 			return "", fmt.Errorf("linkpath must be absolute (got: %q)", ld)
 		}
+		if checkExistence {
+			if _, err := os.Stat(ld); err != nil {
```

Comment:
could you please use `fileutils.Exists()`?

Reply:
Thanks for the review. Fixed to use fileutils.Exists.